hardware/simulation/simulation.mk

```diff
@@ -63,7 +63,7 @@ else
 	bash -c "trap 'make kill-remote-sim' INT TERM KILL; ssh $(SIM_SSH_FLAGS) $(SIM_USER)@$(SIM_SERVER) 'make -C $(REMOTE_ROOT_DIR) sim-build SIMULATOR=$(SIMULATOR) INIT_MEM=$(INIT_MEM) USE_DDR=$(USE_DDR) RUN_EXTMEM=$(RUN_EXTMEM) VCD=$(VCD) TEST_LOG=\"$(TEST_LOG)\"'"
 endif
 
-run:
+run: firmware.bin
```

Comment:
No. When you run simulation the fw is already built. No cross dependencies.

Reply:
Yes, it is already built. But this dependency is here only for copying the firmware.bin to the build folder (icarus, CYCLONEV, etc).